opencti-platform/opencti-graphql/src/domain/user.js

```diff
@@ -686,31 +698,51 @@ export const meEditField = async (context, user, userId, inputs, password = null
   return userEditField(context, user, userId, inputs);
 };
 
-export const deleteAllWorkspaceForUser = async (userId) => {
-  return await elRawDeleteByQuery({
-    index: READ_INDEX_INTERNAL_OBJECTS,
-    refresh: true,
-    body: {
-      query: {
-        bool: {
-          must: [
-            { term: { 'entity_type.keyword': { value: 'Workspace' } } },
-            { term: { 'authorized_members.id.keyword': { value: userId } } }
-          ],
-          filter: {
-            script: {
-              script: 'return (doc[\'authorized_members.id.keyword\'].size() == 1);'
-            }
+const isUserTheLastAdmin = (userId, authorized_members) => {
+  let currentUserIsAdmin = false;
+  let anotherUserIsAdmin = false;
+
+  authorized_members.forEach((oneMember) => {
```

Comment:
Just so you know, this could be done with a .some()
Example:

const currentUserIsAdmin = authorized_members.some(({ id, access_right }) => id === userId && access_right === "admin");

Reply:
oh interesting 👀, thanks !

Reply:
now I want to change this :)

Reply:
done